src/libYARP_sig/src/Vector.cpp

```diff
@@ -41,6 +41,14 @@ class VectorPortContentHeader
 };
 YARP_END_PACK
 
+const std::map<int, std::string> tag2FormatStr = {
+    {BOTTLE_TAG_INT32, "d"},
+    {BOTTLE_TAG_INT64, "ld"},
```

Comment:
I suggest to use [`PRId32` and `PRId64`](https://en.cppreference.com/w/cpp/header/cinttypes) here